lib/Runtime/Library/BuiltIn/BuiltIn.js

```diff
@@ -0,0 +1,60 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+"use strict";
+
+(function (intrinsic) {
+    var platform = intrinsic.BuiltIn;
```

Comment:
>platform [](start = 8, length = 8)

What is the distinction between platform and the global __chakraLibrary objects?

Please add a test to double check that the __chakraLibrary code is not exposed to the user-facing runtime.